mesonbuild/dependencies.py

```diff
@@ -1051,6 +1051,17 @@ def __init__(self, environment, kwargs):
         self.is_found = False
         self.cargs = []
         self.linkargs = []
+        try:
+            pcdep = PkgConfigDependency('sdl2', environment, kwargs)
+            if pcdep.found():
+                self.is_found = True
+                self.cargs = pcdep.get_compile_args()
+                self.linkargs = pcdep.get_link_args()
+                self.version = pcdep.get_version()
+                return
+        except Exception as e:
+            print(e)
```

Comment:
We probably want to do `except DependencyException as e:` and print only the error message so the user knows why `sdl2-config` was used. If some other exception is raised, we probably want it to be raised since it's some other error.